lib/supavisor/helpers.ex

```diff
@@ -371,9 +371,9 @@ defmodule Supavisor.Helpers do
 
   @spec validate_name(String.t()) :: boolean()
   def validate_name(name) do
-    # 1-63 characters, starting with a lowercase letter or underscore, and containing only alphanumeric characters, underscores, and dollar signs. Names with spaces or uppercase letters must be enclosed in double quotes.
+    # 1-63 characters, starting with a lowercase letter or underscore, and containing only alphanumeric characters, underscores, dollar signs, and hyphens. Names with spaces or uppercase letters must be enclosed in double quotes.
     String.length(name) <= 63 and
```

Comment:
Technically we could use `byte_size/1` there, as later we check if the name is ASCII only, so we do not need `String.length/1` there (which is Unicode aware).